src/lib/RuleResultRecombinator.ts

```diff
@@ -24,135 +22,144 @@ export class RuleResultRecombinator {
 		}
 	}
 
-	private static constructCsv([pmdResults]: [string]): string {
-		// TODO: Eventually, we'll need logic to combine disparate result sets together into a single CSV, but for now we
-		//  can proceed with just PMD's results.
-		return this.pmdToCsv(pmdResults);
-	}
-
-	private static constructJunit([pmdResults]: [string]): string {
-		return this.pmdToJunit(pmdResults);
+	private static constructCsv(results: RuleResult[]): string {
+		return this.toCsv(results);
 	}
 
-	private static constructXml([pmdResults]: [string]): string {
-		// TODO: Eventually, we'll need logic to actually combine XMLs together and massage them into the format we want to output,
-		//  but for now we can just return the XML that was provided to us.
-		return pmdResults;
+	private static constructJunit(results: RuleResult[]): string {
+		return this.toJunitXml(results);
```

Comment:
Any reason this needs to invoke another method instead of just doing the logic itself?